src/routes/canvas/toolbar/tools/Eraser.tsx

```diff
@@ -73,14 +77,27 @@ class Eraser extends Tool {
         const eraser = this.state.isPointEraser ? pointEraserProps : eraserProps;
 
         return (
-            <Tool {...this.props} icon={eraser.icon} tool={eraser.tool}>
-                {/* Draw the button to select the other eraser */}
-                {isDoubleClick && <div>
-                    <Button onClick={() => this.setEraser(!this.state.isPointEraser)}>
-                        {this.state.isPointEraser ? eraserProps.icon : pointEraserProps.icon}
-                    </Button>
-                </div>}
-                {isDoubleClick && <SizePicker setSize={this.setSize} />}
+            <Tool {...this.props} icon={<img src={require("../../../../assets/eraser.png")} alt="Icon" />} tool={eraser.tool}>
+                <div className="tool-third-level">
+                    {/* Draw the button to select the other eraser */}
+                    {isDoubleClick &&
+                        <div>
+                            <Button onClick={() => this.setEraser(false)}
```

Comment:
Here the two buttons are defined explicitly. It is a bit of code duplication that could be avoided given that the only difference is the icon used.
Consider refactoring